testing/framework/TestCmd.py

```diff
@@ -1463,6 +1463,8 @@ def file_fixture(self, srcfile, dstfile=None) -> None:
         """
         if is_List(srcfile):
             srcfile = os.path.join(*srcfile)
+        if is_List(dstfile):
```

Comment:
Needed because we're specifying .mod files now?

Reply:
I ran across this because I was following what was documented for the FW, and it didn't work. The place that used it could of course not use it... but it seemed better to finish up for consistency anyway.

Reply:
In fact, here's the failure I still had stashed:
```
Traceback (most recent call last):
  File "/home/mats/github/scons/test/Fortran/USE-MODULE-live.py", line 47, in <module>
    test.file_fixture(['fixture-mod', 'main.f90'], dstfile=['src', 'main.f90'])
  File "/home/mats/github/scons/testing/framework/TestCmd.py", line 1483, in file_fixture
    dstdir, dsttail = os.path.split(dstfile)
                      ^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen posixpath>", line 103, in split
TypeError: expected str, bytes or os.PathLike object, not list
```
So it wasn't specifically about modules, just because I had the temerity to write `dstfile=['src', 'main.f90']` and it didn't handle the list form of destination, unlike for source, which it did handle.